lex.go

```diff
@@ -159,7 +159,6 @@ WriteToBuf:
 // lexSymbolState scans for strings and can emit the following tokens:
 // orToken, andToken, containsToken
 func lexSymbolState(l *lexer) (lexStateFunc, error) {
-	const op = "mql.lexSymbolState"
```

Comment:
Fixes:
```
lex.go:162:8: const op is unused (unused)
	const op = "mql.lexSymbolState"
	      ^

```

Let me know if leaving this in adds any value. Other functions declare an `op` to be used in lexing error strings.

Reply:
I think we can actually remove the `error` from this signature as well, it doesn't look like it's used.